lib/src/auto_injector_base.dart

```diff
@@ -289,16 +290,7 @@ class _AutoInjector extends AutoInjector {
 
   @override
   void addInjector(covariant _AutoInjector injector) {
-    for (final bind in injector._binds) {
-      final index = _binds.indexWhere(
-        (bindElement) => bindElement.className == bind.className,
-      );
-      if (index == -1) {
-        _binds.add(bind);
-      }
-    }
-
-    _paramTransforms.addAll(injector._paramTransforms);
+    _injectorsList.add(injector);
```

Comment:
It's not useful anymore because now we are using the **LayersGraph** to find binds and injectors using Graph with [BFS (Breadth First Search)](https://www.geeksforgeeks.org/breadth-first-search-or-bfs-for-a-graph/).

This list only deserves to help **LayersGraph** fill the `adjacencyList`  when `LayersGraph.initialize()` is called.